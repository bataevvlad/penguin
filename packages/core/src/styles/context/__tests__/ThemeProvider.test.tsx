import {render} from '@testing-library/react-native';

import ThemeProvider from '../ThemeProvider';

describe('ThemeProvider', () => {
  it('render correctly', () => {
    const {toJSON} = render(<ThemeProvider />);
    expect(toJSON()).toMatchSnapshot();
  });
});
