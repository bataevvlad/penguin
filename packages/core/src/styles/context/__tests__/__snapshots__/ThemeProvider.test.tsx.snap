// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeProvider render correctly 1`] = `null`;
